io.unobserved.espansogui.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "io.unobserved.espansoGUI",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
```

Comment:
23.08 is the current runtime

Reply:
I can't remember exactly which package it was, but when trying to build with 23.08 one of the packages required is still only available as a nightly and doesn't have a stable version. Therefore, I went with 22.08 and I will upgrade when there is a stable version available.

Reply:
rust-stable IS available for 23.08